datafusion/core/tests/sqllogictests/postgres/test_files/self_join_with_alias.slt

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+query I
```

Comment:
I realize this has one test per query to mirror the approach in the python test, but I think it would be perfectly fine (and probably much nicer) to put all the queries into the same file

Reply:
I merged queries into fewer files. I tried to find some logic to group them. I can, of course, put all the queries in a single file.